plasmapy/tests/helpers/exceptions.py

```diff
@@ -0,0 +1,109 @@
+"""Exceptions that describe different types of test failures."""
+
+__all__ = [
+    "Failed",
+    "UnexpectedResultError",
+    "InconsistentTypeError",
+    "MissingExceptionError",
+    "UnexpectedExceptionError",
+    "MissingExceptionError",
+    "UnexpectedWarningError",
+    "MissingWarningError",
+    "InvalidTestError",
+    "ExceptionMismatchError",
+    "WarningMismatchError",
```

Comment:
Alphabetize.  Makes the documentation easier to read.  ... There also seems to be duplicates in here. eg `MissingExceptionError`.